ChrisAPI_data.js

```diff
@@ -0,0 +1,88 @@
+
+
+// get todo response object
+async function getTodos() {
+    let fromUrl = "https://jsonplaceholder.typicode.com/todos";// 
+    try {
+        let res = await fetch(fromUrl);
+        return await res.json();
+    } catch (error) {
+        console.log(error); // log
+    }
+}
+
+ function renderTodoById(){
+    //let todoCount = 200;   
```

Comment:
Make sure to clean up your code before checking in. Generally, you are not going to want to keep commented code in unless it's for documentation purposes